<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>730</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>626</width>
    <height>394</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>626</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AMDGPU Colour Manager</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="default_dega">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>580</y>
      <width>88</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>DEGAMMA</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>601</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Colour Transformation Matrix (CTM)</string>
    </property>
    <widget class="QSlider" name="ctm_s_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>160</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-200</number>
     </property>
     <property name="maximum">
      <number>500</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="pageStep">
      <number>10</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="sliderPosition">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::NoTicks</enum>
     </property>
     <property name="tickInterval">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_ctm_s_1">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>31</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>1.00</string>
     </property>
    </widget>
    <widget class="QSlider" name="ctm_s_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>160</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-200</number>
     </property>
     <property name="maximum">
      <number>500</number>
     </property>
     <property name="pageStep">
      <number>10</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="sliderPosition">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_ctm_s_2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>60</y>
       <width>31</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>0.00</string>
     </property>
    </widget>
    <widget class="QSlider" name="ctm_s_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>160</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-200</number>
     </property>
     <property name="maximum">
      <number>500</number>
     </property>
     <property name="pageStep">
      <number>10</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="sliderPosition">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_ctm_s_3">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>90</y>
       <width>31</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>0.00</string>
     </property>
    </widget>
    <widget class="QSlider" name="ctm_s_4">
     <property name="geometry">
      <rect>
       <x>205</x>
       <y>30</y>
       <width>160</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-200</number>
     </property>
     <property name="maximum">
      <number>500</number>
     </property>
     <property name="pageStep">
      <number>10</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="sliderPosition">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_ctm_s_4">
     <property name="geometry">
      <rect>
       <x>365</x>
       <y>30</y>
       <width>31</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>0.00</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_ctm_s_5">
     <property name="geometry">
      <rect>
       <x>365</x>
       <y>60</y>
       <width>31</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>1.00</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_ctm_s_6">
     <property name="geometry">
      <rect>
       <x>365</x>
       <y>90</y>
       <width>31</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>0.00</string>
     </property>
    </widget>
    <widget class="QSlider" name="ctm_s_5">
     <property name="geometry">
      <rect>
       <x>205</x>
       <y>60</y>
       <width>160</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-200</number>
     </property>
     <property name="maximum">
      <number>500</number>
     </property>
     <property name="pageStep">
      <number>10</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="sliderPosition">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="ctm_s_6">
     <property name="geometry">
      <rect>
       <x>205</x>
       <y>90</y>
       <width>160</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-200</number>
     </property>
     <property name="maximum">
      <number>500</number>
     </property>
     <property name="pageStep">
      <number>10</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="sliderPosition">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="ctm_s_7">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>30</y>
       <width>160</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-200</number>
     </property>
     <property name="maximum">
      <number>500</number>
     </property>
     <property name="pageStep">
      <number>10</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="sliderPosition">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_ctm_s_7">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>30</y>
       <width>31</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>0.00</string>
     </property>
    </widget>
    <widget class="QSlider" name="ctm_s_8">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>60</y>
       <width>160</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-200</number>
     </property>
     <property name="maximum">
      <number>500</number>
     </property>
     <property name="pageStep">
      <number>10</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="sliderPosition">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_ctm_s_8">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>60</y>
       <width>31</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>0.00</string>
     </property>
    </widget>
    <widget class="QSlider" name="ctm_s_9">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>90</y>
       <width>160</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-200</number>
     </property>
     <property name="maximum">
      <number>500</number>
     </property>
     <property name="pageStep">
      <number>10</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="sliderPosition">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_ctm_s_9">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>90</y>
       <width>31</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>1.00</string>
     </property>
    </widget>
    <widget class="QPushButton" name="default_ctm">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>120</y>
       <width>88</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Default</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>471</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>Saturation</string>
     </property>
     <widget class="QSlider" name="sat_s">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>391</width>
        <height>31</height>
       </rect>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>300</number>
      </property>
      <property name="pageStep">
       <number>9</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_sat">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>20</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>100%</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>601</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Gamma Settings</string>
    </property>
    <widget class="QSlider" name="rega_s_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>160</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>300</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="pageStep">
      <number>10</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="sliderPosition">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::NoTicks</enum>
     </property>
     <property name="tickInterval">
      <number>0</number>
     </property>
    </widget>
    <widget class="QSlider" name="rega_s_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>30</y>
       <width>160</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>300</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="pageStep">
      <number>10</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="sliderPosition">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::NoTicks</enum>
     </property>
     <property name="tickInterval">
      <number>0</number>
     </property>
    </widget>
    <widget class="QSlider" name="rega_s_3">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>30</y>
       <width>160</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>300</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="pageStep">
      <number>10</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="sliderPosition">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::NoTicks</enum>
     </property>
     <property name="tickInterval">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_rega_s_1">
     <property name="geometry">
      <rect>
       <x>173</x>
       <y>30</y>
       <width>31</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>1.00</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_rega_s_2">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>30</y>
       <width>31</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>1.00</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_rega_s_3">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>30</y>
       <width>41</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>1.00</string>
     </property>
    </widget>
    <widget class="QPushButton" name="default_rega">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>80</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Default</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>191</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>Gamma Mode</string>
     </property>
     <widget class="QRadioButton" name="degam_r_1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>SRGB</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="degam_r_2">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>30</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Linear</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QSlider" name="hue_s">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>480</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_v_1">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>370</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_v_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>390</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_v_3">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>410</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_sv_1">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>380</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_sv_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>400</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_sv_3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>420</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>440</y>
      <width>88</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <zorder>groupBox</zorder>
   <zorder>default_dega</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>hue_s</zorder>
   <zorder>label_v_1</zorder>
   <zorder>label_v_2</zorder>
   <zorder>label_v_3</zorder>
   <zorder>label_sv_1</zorder>
   <zorder>label_sv_2</zorder>
   <zorder>label_sv_3</zorder>
   <zorder>pushButton</zorder>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
